interface/main/finder/multi_patients_finder.php

```diff
@@ -0,0 +1,275 @@
+<?php
+
+/**
+ * Multi select patient.
+ *
+ * Copyright (C) 2016 Amiel Elboim <amielel@matrix.co.il>
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @author  Amiel Elboim <amielel@matrix.co.il>
+ * @link    http://www.open-emr.org
+ */
+
+require_once('../../globals.php');
+require_once("$srcdir/patient.inc");
+use OpenEMR\Core\Header;
+
+// for editing selected patients
+if (isset($_GET['patients'])) {
+    $patients = rtrim($_GET['patients'], ";");
+    $patients = explode(';', $patients);
+    $results = array();
+    foreach ($patients as $patient) {
+        $result=getPatientData($patient, 'id, pid, lname, fname, mname, pubpid, ss, DOB, phone_home');
+        //clean data using 'text' function
+        $result=array_map('text', $result);
+        $result['DOB'] = oeFormatShortDate($result['DOB']);
```

Comment:
Also best to only call oeFormatShortDate when it is outputted since any algorithms within script should use he YYYY-MM--DD format.